src/ec/packages/ec-component-language-list/demo/data.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable import/no-extraneous-dependencies */
+export const common = {
+  iconPath: 'static/icons.svg',
+};
+
+export const items = [
```

Comment:
why don't you use the data from the specs? https://github.com/ec-europa/europa-component-library/blob/v2-dev/src/systems/ec/specs/components/language-list/demo/data--overlay.js

Reply:
the structure is different (for instance for link and icons, there is an additional nested level, and field names are different).
So I found it easier that way

Reply:
Ideally, we should write down the content specs and tests in the component specs package and use them at the twig component level in order to check if the implementation is okay. Which means we probably need to write a transform function at the twig test level.

`twig_component(twig_test_transform(specs_demo1_data)) = specs_test_demo1_markup`